tools/ci_build/github/apple/c/onnxruntime-mobile-c.podspec.template

```diff
@@ -7,19 +7,19 @@ Pod::Spec.new do |spec|
     spec.source                 = { :http => "file:///http_source_placeholder" }
     spec.summary                = "ONNX Runtime Mobile C/C++ Pod"
     spec.platform               = :ios, "@IOS_DEPLOYMENT_TARGET@"
-    spec.vendored_frameworks    = "onnxruntime.framework"
+    spec.vendored_frameworks    = "onnxruntime.xcframework"
     spec.static_framework       = true
     spec.weak_framework         = [ @WEAK_FRAMEWORK@ ]
-    spec.source_files           = "onnxruntime.framework/Headers/*.h"
+    spec.source_files           = "Headers/*.h"
     spec.preserve_paths         = [ @LICENSE_FILE@ ]
     spec.description            = <<-DESC
     A pod for the ONNX Runtime Mobile C/C++ library.
                                     DESC
-    spec.library                 = "c++"
-    spec.pod_target_xcconfig     = {
+    spec.library                = "c++"
+    spec.pod_target_xcconfig    = {
         "OTHER_CPLUSPLUSFLAGS" => "-fvisibility=hidden -fvisibility-inlines-hidden",
     }
-    spec.user_target_xcconfig    = {
+    spec.xcconfig               = {
```

Comment:
Had to make this change, otherwise the `pod lib lint` will fail since we don't include arm64 ios-simulator yet, hopefully this can be removed after we enable the arm64 ios-simulator build

Reply:
is there documentation for the xcconfig attribute? I didn't see it here https://guides.cocoapods.org/syntax/podspec.html

Reply:
I found this on stackoverflow, this is not in the official podspec syntax, we should be able to get rid of this soon

Reply:
This is no longer needed after the latest commits

Reply:
nice :)